content/en/docs/hertz/reference/json.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "JSON Marshal Library"
+linkTitle: "JSON Marshal Library"
+weight: 3
+description: >
+
+---
+
+
+Hertz integrates [sonic](https://github.com/bytedance/sonic) by default as a json marshal library for serializing data with the ctx.JSON interface. sonic is an ultra-high performance golang json library, see Sonic [README](https://github.com/bytedance/sonic) for details.
```

Comment:
1. add code block for `ctx.JSON`
2. sonic -> Sonic 
3. please fix those in zh doc as well

Reply:
done